scss/index.scss

```diff
@@ -1,5 +1,11 @@
 // our project themes
-@import "./theme/colors";
+@import './theme/colors';
 
 // import your custom scss files below here
-@import "./heroNotificationBox.scss";
+@import './heroNotificationBox.scss';
```

Comment:
In general, try not to format code not touched by the PR. I assume this was an auto-formatter

Reply:
I added an scss file that's why I formatted it 😅